### PR TITLE
Spin out tools

### PR DESCRIPTION
**Pull request by wl2002**
Basics are working, need to fix comments and do more testing to make sure it's all the same as before. Getting debug messages while running each tool, will figure out how to make those go away unless someone has any quick ideas?

### CODE REVIEW

**Comment by wl2002** [comment]
Figured out debug messages - will fix tomorrow

**Comment by kisom** [comment]
Debug messages: you'll need to set `log.Level` to something other than `log.LevelDebug`. We'll need to look at moving the -loglevel flag to the `cli` package, instead of the top-level `cfssl` binary. The downside to this is that it will break scripts/configuration management/systemd unit files that are relying on the log level flag preceeding the command.

The tools should probably be prefaced with a "cfssl-"; for example, renaming the "cmd/bundle" directory to "cmd/cfssl-bundle". Go will use the directory name as the executable name, and it's a good idea here, I think, to namespace our commands. This is the same approach git takes, for example.

**Comment by wl2002** [comment]
Files/directory names changed as requested. Ready for review

File: cmd/cfssl-bundle/cfssl-bundle.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+	Usage of bundle:
+		- Bundle local certificate files
+        	bundle -cert file [-ca-bundle file] [-int-bundle file] [-int-dir dir] [-metadata file] [-key keyfile] [-flavor optimal|ubiquitous|force] [-password password]
+		- Bundle certificate from remote server.
+        	bundle -domain domain_name [-ip ip_address] [-ca-bundle file] [-int-bundle file] [-int-dir dir] [-metadata file]
+
+	Flags:
+	`
+	flag.IntVar(&log.Level, "loglevel", log.LevelInfo, "Log level")
```
Comment (kisom) [inline cmd/cfssl-bundle/cfssl-bundle.go]: You should move this to `registerFlags`.

File: cmd/cfssl-certinfo/cfssl-certinfo.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+	Usage of certinfo:
+		- Data from local certificate files
+        	certinfo -cert file
+		- Data from certificate from remote server.
+        	certinfo -domain domain_name
+
+	Flags:
+	`
+	flag.IntVar(&log.Level, "loglevel", log.LevelInfo, "Log level")
```
Comment (kisom) [inline cmd/cfssl-certinfo/cfssl-certinfo.go]: You should move this to `registerFlags`.

File: cmd/cfssl-newkey/cfssl-newkey.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+	Usage of genkey:
+        	newkey CSRJSON
+
+	Arguments:
+        	CSRJSON:    JSON file containing the request, use '-' for reading JSON from stdin
+
+	Flags:
+	`
+	flag.IntVar(&log.Level, "loglevel", log.LevelInfo, "Log level")
```
Comment (kisom) [inline cmd/cfssl-newkey/cfssl-newkey.go]: You should move this to `registerFlags`.

File: cmd/cfssl-scan/cfssl-scan.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	var usageText = `cfssl-scan -- scan a host for issues
+		Usage of scan:
+        	scan [-family regexp] [-scanner regexp] [-timeout duration] [-ip IPAddr] HOST+
+        	scan -list
+
+		Arguments:
+        	HOST:    Host(s) to scan (including port)
+		Flags:
+		`
+	flag.IntVar(&log.Level, "loglevel", log.LevelInfo, "Log level")
```
Comment (kisom) [inline cmd/cfssl-scan/cfssl-scan.go]: You should move this to `registerFlags`.

**Comment by wl2002** [comment]
Fixed!

**Comment by kisom** [comment]
LGTM.

**Comment by kisom** [comment]
Can you squash this into a single commit?

**Comment by wl2002** [comment]
Done :)

**Comment by kisom** [comment]
LGTM.

**Pull request merged**